api/lib/searchOperators.js

```diff
@@ -132,11 +189,12 @@ module.exports = {
         parameters: [modifiedSearchQuery],
       };
     }
-    return {
+    const result = {
```

Comment:
nitpick: Do we know why is this called "result"? it seems like a query.

Reply:
Yeah, changed it to queryObj